test/broadcast.jl

```diff
@@ -376,6 +376,7 @@ end
 
 # Check that broadcast!(f, A) populates A via independent calls to f (#12277, #19722).
```

Comment:
Perhaps expand the test description with a note on `broadcast!(f, A, x::Number...)` and reference this pull request?